jablib/src/test/java/org/jabref/logic/layout/LayoutEntryTest.java

```diff
@@ -73,32 +73,38 @@ public String layout(String layoutFile, BibEntry entry) throws IOException {
         return layout.doLayout(entry, null);
     }
 
-    @Test
-    void parseMethodCalls() {
-        assertEquals(1, LayoutEntry.parseMethodsCalls("bla").size());
-        assertEquals("bla", LayoutEntry.parseMethodsCalls("bla").getFirst().getFirst());
-
-        assertEquals(1, LayoutEntry.parseMethodsCalls("bla,").size());
-        assertEquals("bla", LayoutEntry.parseMethodsCalls("bla,").getFirst().getFirst());
-
-        assertEquals(1, LayoutEntry.parseMethodsCalls("_bla.bla.blub,").size());
-        assertEquals("_bla.bla.blub", LayoutEntry.parseMethodsCalls("_bla.bla.blub,").getFirst().getFirst());
-
-        assertEquals(2, LayoutEntry.parseMethodsCalls("bla,foo").size());
-        assertEquals("bla", LayoutEntry.parseMethodsCalls("bla,foo").getFirst().getFirst());
-        assertEquals("foo", LayoutEntry.parseMethodsCalls("bla,foo").get(1).getFirst());
+    @ParameterizedTest
+    @CsvSource(textBlock = """
+            bla , bla
+            bla , bla
+            _bla.bla.blub , _bla.bla.blub
+            """)
```

Comment:
This is more readable, but reads strange as there is a space before the comma

What about

    bla, bla

? This would make it more close to "natural" sentences.